src/components/ContentWidgets/ToDoList/TodoListEditor.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { TodoItem, TodoList } from './TodoList';
+import "./TodoStyle.css";
+
+// an input box for adding a new todo item or checking/unchecking
+// existing todo items.
+//
+// Last Modified
+// Max Zhou
+// July 6, 2019
+//
+export const TodoListEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+	// get the list
+	let currentList: TodoItem[] = editedContent.todoList_items || [];
+
+	let list_html = currentList.map((item: TodoItem, index: number) => <li className={item.check} key={index}>{item.name}<button type="button" onClick={(e) => handleCheck(index)}>done</button></li>);
```

Comment:
stick to camel casing variables, `listHtml` should be used here instead. It actually shouldn't even be `listHtml`, it should be something like `listItems`

Reply:
You don't use the `e` in the `onClick` handler, so just remove it.

Reply:
This can potentially crash because `handleCheck` has not been defined yet.

Reply:
Keep lines to under 100 characters